h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.component.html

```diff
@@ -65,17 +65,6 @@
          target="_blank">
         <clr-icon shape="help" class="is-info"></clr-icon>
       </a>
-      <clr-alert *ngIf="errorFlag"
```

Comment:
What's the purpose of the changes in this file?

Reply:
We should've filed a separate issue for this, but it's a cosmetic issue where the Clarity warning box, if any error is present, was appearing twice in the view. It's a bit of an excuse to say we wanted to sneak in some small stuff as we go, but will improve next time.